src/components/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { IButton, EIconPosition } from './Button.model';
+import {
+  FillButton,
+  OutlineButton,
+  Image,
+  DisableButton,
+  TransparentButton
+} from './Button.styled';
+
+const Button = ({
+  children,
+  icon = undefined,
+  iconPos = EIconPosition.NONE,
+  type = 'fill'
+}: IButton) => {
+  const content = () => {
+    if (icon && iconPos === EIconPosition.NONE) {
+      throw Error('You need to specify icon position (iconPos prop)');
```

Comment:
Dont you think that "Crashing page" is that so good idea to notify user that he need to specify some props ?